From 7eee4e0f8639e52e03c003e7415ab20d36c8abbd Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sat, 25 Aug 2018 20:31:39 -0600
Subject: [PATCH 16/16] msm8974-common: Allow additional gralloc 1.0 buffer
 usage bits

* Our platform requires 0x02000000U. Per headers from exynos5420,
  this appears to be USAGE_PRIVATE_NONSECURE.

Change-Id: Ie5a8a46fef1c0f28ebd2cd3b3d29cc19763bc2a3
---
 BoardConfigCommon.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index e115f0e..7109ee2 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -56,6 +56,7 @@ endif
 # Display
 NUM_FRAMEBUFFER_SURFACE_BUFFERS := 3
 OVERRIDE_RS_DRIVER:= libRSDriver_adreno.so
+TARGET_ADDITIONAL_GRALLOC_10_USAGE_BITS := 0x02000000U
 
 # Shader cache config options
 # Maximum size of the  GLES Shaders that can be cached for reuse.
-- 
2.17.1

